tortoise/tests/test_basic.py

```diff
@@ -0,0 +1,19 @@
+from tortoise.contrib.test import TestCase
+from tortoise.tests.testmodels import Tournament
+
+
+class TestBasic(TestCase):
+    async def test_basic(self):
+        event = await Tournament.create(name='Test')
+        await Tournament.filter(id=event.id).update(name='Updated name')
+        saved_event = await Tournament.filter(name='Updated name').first()
+        self.assertEquals(saved_event.id, event.id)
+        await Tournament(name='Test 2').save()
+        self.assertEquals(
+            await Tournament.all().values_list('id', flat=True),
+            [1, 2]
+        )
+        self.assertEquals(
+            await Tournament.all().values('id', 'name'),
+            [{'id': 1, 'name': 'Updated name'}, {'id': 2, 'name': 'Test 2'}]
+        )
```

Comment:
Regarding `Record` issue - currently `values` returns records for asyncpg and I don't want to change this behaviour because it will be adding overhead to parse highly efficient asyncpg `Record` to dict just for the matter of convenience  for display. I think you could parse them into dicts here in test.

Reply:
I get your point.
The point of an ORM to me is to isolate much of the differences between Database implementation, and to provide convenience.
I think we need to be careful and have a good reason to change the behaviour from standard.

The test is testing what is expected, so I don't think it should change.
Possibly have a `.records()` method, that gives `Record` (or Record-like) objects in response, and call it an optimisation?

Reply:
Ok, I read up on `Record` and I see, it is a dict-like tuple object.
So, almost like a namedtuple. It is in general more memory efficient as it is a tuple at heart.

Reply:
It makes me think, for all RDBMS you know the order of the returned fields, so this is an opportunity for global optimisation, as we can get rid of dicts internally.
This could noticeably reduce memory pressure, and hence performance under load.
But I think is out of scope of this.

For now I'll make the ValuesQuery do the change itself, so the rest of asyncpg can keep on using the more optimal format.